### PR TITLE
Display a toast notification when a plan execution completes

### PR DESCRIPTION
**Pull request by AllenBW**
same as #247 cept into the alpha branch rather than sprint7

### CODE REVIEW

**Comment by priley86** [comment]
i decided to name sprint8 branch `gaprindashvili²`

**Comment by AllenBW** [comment]
we're good to go on this one!

File: app/javascript/react/screens/App/Overview/Overview.js [hunk 1]
```diff
@@ -82,13 +91,51 @@ class Overview extends React.Component {
...
+      const oldMigrationIds = prevProps.finishedTransformationPlans.map(
+        plan => plan.id
+      );
+      const freshMigrations = finishedTransformationPlans.filter(
+        migration => !oldMigrationIds.includes(migration.id)
+      );
+
+      freshMigrations.forEach(plan => {
+        const planStatus =
+          plan.miq_requests[0].status.toLowerCase() === 'complete';
```
Comment (michaelkro) [inline app/javascript/react/screens/App/Overview/Overview.js]: oops, missed this my first time through.  A plan can have more than one request, for example, if a plan's first request fails, and then the user retries, the plan will have a total of 2 requests

We need to do something like
```javascript
const [mostRecentRequest] = plan.miq_requests.slice(-1)
```

**Comment by priley86** [comment]
thanks 👍 - is the scrolling issue fixed?

**Comment by priley86** [comment]
so can these existing [inline styles](https://github.com/priley86/miq_v2v_ui_plugin/blob/master/app/javascript/react/screens/App/Overview/Overview.js#L80) now be removed?

**Comment by AllenBW** [comment]
Oh if we encapsulate it in a fragment rather than a div, yeah i believe so!

File: app/javascript/react/screens/App/Overview/Overview.js [hunk 2]
```diff
@@ -182,10 +230,7 @@ class Overview extends React.Component {
     );
 
     const overviewContent = (
-      <div
-        className="row cards-pf"
-        style={{ overflow: 'auto', paddingBottom: 1, height: '100%' }}
-      >
+      <React.Fragement className="row cards-pf">
```
Comment (michaelkro) [inline app/javascript/react/screens/App/Overview/Overview.js]: :) `Fragement`

Comment (michaelkro) [reply app/javascript/react/screens/App/Overview/Overview.js]: But, it looks like we actually still need those inline styles

**Comment by priley86** [comment]
ok - i'm good this one for now. Filed [265](https://github.com/priley86/miq_v2v_ui_plugin/issues/265) for future integration to the Detail view...

nice job @AllenBW 😄

**Pull request merged**